subprojects/file-collections/src/main/java/org/gradle/api/internal/file/collections/FilteredMinimalFileTree.java

```diff
@@ -62,11 +60,6 @@ public MinimalFileTree filter(PatternFilterable patterns) {
     @Override
     public void visitStructure(MinimalFileTreeStructureVisitor visitor, FileTreeInternal owner) {
         tree.visitStructure(new MinimalFileTreeStructureVisitor() {
-            @Override
-            public FileCollectionStructureVisitor.VisitType prepareForVisit(FileCollectionInternal.Source source) {
```

Comment:
Why can this be removed?

Reply:
Only the old continuous build used this method, so the minimal file trees could call back. Now that this doesn't need to happen any more, there is no need for the `prepareForVisit` method.

Reply:
I see it's removed from the interface